gateway-ha/src/test/java/io/trino/gateway/ha/router/TestRoutingGroupSelectorExternal.java

```diff
@@ -84,7 +84,7 @@ static Stream<RulesExternalConfiguration> provideRoutingRuleExternalConfig()
     {
         RulesExternalConfiguration restConfig = new RulesExternalConfiguration();
         restConfig.setUrlPath("http://localhost:8080/api/public/gateway_rules");
-        restConfig.setBlackListHeaders(new ArrayList<>(List.of("Authorization")));
+        restConfig.setExcludeHeaders(new ArrayList<>(List.of("Authorization")));
```

Comment:
Why are we calling `new ArrayList` here? Also, why does this test class use `@ParameterizedTest` just for single case?

Reply:
> Why are we calling new ArrayList here? 

True a list is sufficent.

> Also, why does this test class use @ParameterizedTest just for single case?

what do you mean by a single case?

Reply:
I meant this method returns a single `RulesExternalConfiguration`.

Reply:
I am not that familiar with Junit 5 so please bear with me.

Well, I wanted to reduce code duplication for configs, as it could be used several times. 
And i thought this way of using parameterized test makes it concise to test.

But now that you mention, since this is not complex and returns as a single object,
i could simply create this object in initialize() function, make it where it is `@BeforeAll`. right?

Reply:
RulesExternalConfiguration is a mutable object. Please don't put it in the fields. 
We can simply create a helper method and use it in each test.

Reply:
I pushed the small change.